spec.html

```diff
@@ -25353,14 +25380,21 @@ <h1>Function.prototype</h1>
     <!-- es6num="19.2.3" -->
     <emu-clause id="sec-properties-of-the-function-prototype-object">
       <h1>Properties of the Function Prototype Object</h1>
-      <p>The Function prototype object is the intrinsic object <dfn>%FunctionPrototype%</dfn>. The Function prototype object is itself a built-in function object. When invoked, it accepts any arguments and returns *undefined*. It does not have a [[Construct]] internal method so it is not a constructor.</p>
+      <p>The Function prototype object:</p>
+      <ul>
+        <li>is the intrinsic object <dfn>%FunctionPrototype%</dfn>.</li>
+        <li>is itself a built-in function object.</li>
+        <li>accepts any arguments and returns *undefined* when invoked.</li>
+        <li>does not have a [[Construct]] internal method; it cannot be used as a constructor with the `new` operator.</li>
+        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %ObjectPrototype%.</li>
+        <li>has an [[Extensible]] internal slot whose initial value is *true*.</li>
```

Comment:
Not related to this PR; just dropping a note to myself for later: does this need a specific call-out, or is the default [[Extensible]] true?

Reply:
Prose-wise, [Clause 17](https://tc39.github.io/ecma262/#sec-ecmascript-standard-built-in-objects) says `Unless specified otherwise, the [[Extensible]] internal slot of a built-in object initially has the value *true*.` And algorithm-wise, `CreateIntrinsics` creates `%FunctionPrototype%` via `CreateBuiltinFunction`, which explicitly sets `[[Extensible]]` to `*true*`.

So yeah, that specific point is redundant.

(And similarly for the `[[Extensible]]` bullet of 13 other intrinsics. Offhand, it looks like the only intrinsic with `[[Extensible]] = *false*` is `%ThrowTypeError%`, although interestingly, the prose that says so doesn't agree with the setting in CreateBuiltinFunction. CreateIntrinsics should maybe `Set _thrower_.[[Extensible]] to *false*` explicitly.)

Reply:
Is it worth removing the redundant points in this PR?

Reply:
I don't know, some people might like them. Anyhow, the question is orthogonal to this PR. (But I'd be fine with submitting a separate commit if there's consensus to remove them.)